src/src/Controller/CalculatorController.php

```diff
@@ -38,13 +38,13 @@ public function index(Request $request): Response
                         }
                         break;
                     default:
-                        $result = 'Invalid operation';
+                        $result = 'Invalid operator';
                 }
             } else {
                 $result = 'Invalid input';
             }
     
-            dump($result);
+            return $this->redirectToRoute('app_calculator', ['result' => $result]);
```

Comment:
POST requests need to end with a redirect to avoid multiple submissions (not a big issue for this calculator, but the rule is applied in general in the framework). We're passing the result as a query parameter so we can display it after the redirect.